src/Castle.Windsor/Castle.Windsor-VS2017.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Castle.Windsor</Description>
+    <TargetFramework>net45</TargetFramework>
+    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+    <NoWarn>$(NoWarn);1591</NoWarn>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <AssemblyName>Castle.Windsor</AssemblyName>
+    <RootNamespace>Castle</RootNamespace>
+      <AssemblyOriginatorKeyFile>../../buildscripts/CastleKey.snk</AssemblyOriginatorKeyFile> 
+     <SignAssembly>true</SignAssembly> 
+    <PackageId>Castle.Windsor</PackageId>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Castle.Core" Version="3.3.0.0" />
```

Comment:
It seems a little strange to include the extra ".0" when the nuget version doesn't have it.